### PR TITLE
feat: #351 Add styles

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: mw-webapp/src/logic/wayPage/WayPage.module.scss [hunk 1]
```diff
@@ -82,3 +82,7 @@
     margin-bottom: $defaultMargin;
   }
 }
+
+.goalSuSection {
+  width: 30%;
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/WayPage.module.scss:87]: are you sure?

File: mw-webapp/src/logic/wayPage/WayPage.tsx [hunk 2]
```diff
@@ -487,7 +487,7 @@ export const WayPage = (props: WayPageProps) => {
             className={styles.goalDescription}
           />
         </div>
-        <div>
+        <div className={styles.goalSuSection}>
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/WayPage.tsx:490]: what is suSections?

File: mw-webapp/src/logic/wayPage/WayStatistic.tsx [hunk 3]
```diff
@@ -105,9 +105,9 @@ export const WayStatistic = (props: WayStatisticProps) => {
 
   const lastCalendarTwoWeekTotalTime = lastTwoWeekJobsAmount.reduce((totalTime, jobDone) => totalTime + jobDone.time, 0);
 
-  const lastCalendarTwoWeekAverageWorkingTime = Math.round(lastCalendarTwoWeekTotalTime / AMOUNT_DAYS_IN_WEEK);
+  const lastCalendarTwoWeekAverageWorkingTime = Math.round(lastCalendarTwoWeekTotalTime / AMOUNT_DAYS_IN_TWO_WEEK);
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/WayStatistic.tsx:108]: what if there is only 9 days in the report? for example?

File: mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss [hunk 4]
```diff
@@ -38,11 +38,11 @@ $listLeftMargin: 20px;
   margin-bottom: $defaultMargin;
 }
 
-.comments li {
+.comments > li {
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss:41]: could we use specific class name for lists?

File: mw-webapp/src/utils/ThemeWorker.ts [hunk 5]
```diff
@@ -73,7 +73,8 @@ export class ThemeWorker {
    */
   public static loadTheme() {
     const theme = localStorageWorker.getItemByKey("theme");
-    this.setTheme(theme ?? DEFAULT_THEME);
+    // TODO: font use as
```
Comment (tritonJS826) [inline mw-webapp/src/utils/ThemeWorker.ts:76]: fondt -> dont

Comment (tritonJS826) [reply mw-webapp/src/utils/ThemeWorker.ts:76]: emm but there wasnt 'as' operator before, why did you add it?

File: mw-webapp/src/utils/LanguageWorker.ts [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    localStorageWorker.setItemByKey("language", language);
+  }
+
+  /**
+   * Load language
+   */
+  public static loadLanguage() {
+    const language = localStorageWorker.getItemByKey("language");
+    // TODO: font use as
+    this.setLanguage(language as Language ?? DEFAULT_LANGUAGE);
```
Comment (tritonJS826) [inline mw-webapp/src/utils/LanguageWorker.ts:31]: the same here. why do we need as operator?

File: mw-webapp/src/utils/LanguageWorker.ts [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+/**
+ * Available languages
+ */
+export enum Language {
+  ENGLISH = "english",
+  RUSSIAN = "russian"
+}
+
+export const DEFAULT_LANGUAGE = Language.ENGLISH;
```
Comment (tritonJS826) [inline mw-webapp/src/utils/LanguageWorker.ts:11]: let's add a comment to public member

File: mw-webapp/src/logic/wayPage/WayPage.module.scss (hunk 1 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/WayPage.module.scss:85]: yes

File: mw-webapp/src/logic/wayPage/WayPage.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/WayPage.tsx]: subcection

File: mw-webapp/src/logic/wayPage/WayStatistic.tsx (hunk 3 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/WayStatistic.tsx:108]: So? average working time allways about 2 weeks. Even if you just started your way

File: mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss (hunk 4 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/reportsTable/WayColumns.module.scss:41]: maybe

File: mw-webapp/src/utils/ThemeWorker.ts (hunk 5 above)
Comment (Ekaterina1994) [reply mw-webapp/src/utils/ThemeWorker.ts]: deleted

File: mw-webapp/src/utils/LanguageWorker.ts (hunk 6 above)
Comment (Ekaterina1994) [reply mw-webapp/src/utils/LanguageWorker.ts]: deleted

File: mw-webapp/src/utils/LanguageWorker.ts (hunk 7 above)
Comment (Ekaterina1994) [reply mw-webapp/src/utils/LanguageWorker.ts:11]: done

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/logic/wayPage/WayStatistic.tsx (hunk 3 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/WayStatistic.tsx]: Fixed

File: mw-webapp/src/logic/wayPage/WayStatistic.tsx [hunk 8]
```diff
@@ -14,6 +14,11 @@ interface WayStatisticProps {
    * Reports of specific way
    */
   dayReports: DayReport[];
+
+  /**
+   * Date of way created
+   */
+  wayCreated: Date;
```
Comment (tritonJS826) [inline mw-webapp/src/logic/wayPage/WayStatistic.tsx:21]: is it convinietnNaming? didn't we use 'createdAt'?

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/logic/wayPage/WayStatistic.tsx (hunk 8 above)
Comment (Ekaterina1994) [reply mw-webapp/src/logic/wayPage/WayStatistic.tsx]: renamed

**Pull request merged**